packages/cascara/src/ui/Stat/__globals.js

```diff
@@ -0,0 +1,16 @@
+import pt from 'prop-types';
+
+export const PROP_TYPES = {
+  /** Stats can have their own css class name */
+  className: pt.string,
+  /** Stats can be fuild */
+  fluid: pt.bool,
+  /** Stats can have a label */
+  label: pt.string,
+  /** Stats can be clickable */
+  onClick: pt.func,
+  /** Stats can have a sub text */
+  sub: pt.oneOfType([pt.number, pt.string]),
+  /** Stats can have a value */
+  value: pt.oneOfType([pt.number, pt.string]),
```

Comment:
We have these prop types being defined in two places. Which is our source of truth?

Reply:
The one in the component. Props were initially put here but Docs doesn't support showing the props table that isn't part of the component file.

Reply:
@lordpixel if we are not using these, let's remove them then. Leaving them is going to confuse someone coming along who might see these and try to change them. If we are forced for now to use the propTypes inside the file, we should keep those instead of these.

I need to look at React Docgen to see if they have the updates yet that will allow us to compose external props.

Reply:
@brianespinosa it makes sense, I forgot to remove the declaration in the globals when I realized that those weren't picked up by the propsTable component. 👍